server/helpers/dates.js

```diff
@@ -1,7 +1,10 @@
 import format from 'date-fns/format';
+import formatISO from 'date-fns/formatISO';
 
 export const getTodaysDateInWords = () => format(new Date(), 'd MMM yyyy');
 
 export const getTodaysDateShortCode = () => format(new Date(), 'ddMMyyyy');
 
 export const getTodaysDateStandard = () => format(new Date(), 'yyyy-MM-dd');
+
+export const getDateWithTimestamp = () => formatISO(new Date(), 'yyyy-MM-dd');
```

Comment:
I don't think 'yyyy-MM-dd' has no effect on this function. 

```suggestion
export const getDateWithTimestamp = () => formatISO(new Date());
```